### PR TITLE
Show keyring id problem

### PR DESCRIPTION
**Pull request by rudi-cilibrasi**
i don't understand why the keyringentry `.id` is not available here.

### CODE REVIEW

File: src/prom/boot.mjs [hunk 1]
```diff
@@ -67,14 +67,20 @@ async function promBootStageOne(dispatch, getState) {
   await facadeGetAddresses({dispatch, getState});
 }
 
+async function bootBuildScopes({/* dispatch, */ getState}) {
+  const chainIdList = transientChainIdListSelector(getState());
+  console.log('CHAINIDLIST', chainIdList);
+  const keyringIdList = userProfileManagerSelector(getState()).profile.keyring.getEntries()[0];
```
Comment (ethanfrey) [inline src/prom/boot.mjs]: Problem is that you are using old (private) functions.

`...profile.entryIds.value` is what you want. You *should not* use `profile.keyring` anywhere if you can avoid it. This is unsupported abuse of the API. All features I add are in public api.

Comment (ethanfrey) [reply src/prom/boot.mjs]: Okay, the bigger problem is that when I released late the other night, I forgot to run `yarn build` first, so the javascript on npm was not the most recent :( I thought this happened automatically in `npm publish`....

I will have to rebuild and release 0.5.3... hmmmm....

**Comment by ethanfrey** [comment]
Okay, I took the liberty of experimenting with this branch and fixing it.
It now is updated to iov-core v0.5.3 and the console logs show the id is properly working now.

Feel free to make any other changes as needed, and then merge it, so all functionality is now properly exposed. Sorry about the v0.5.2 npm mishap.

**Comment by rudi-cilibrasi** [comment]
you know what, @ethanfrey ?  i feel the wind beneath my wings, the cool breeze blowing through my hair, and i am now in alignment and ready. i feel it is time to merge.

**Pull request merged**